hydra-rdp.c

```diff
@@ -10,7 +10,7 @@
 #include "hydra-mod.h"
 
 extern char *HYDRA_EXIT;
-#ifndef LIBFREERDP2
+#if !defined(LIBFREERDP2) || (LIBFREERDP3)
```

Comment:
if the library versions make no different is v2 or v3 I would just name it LIBFREERDP

Reply:
Agreed, just fixed this